src/test/csfle.rs

```diff
@@ -3220,6 +3216,7 @@ async fn range_explicit_encryption_test(
         KV_NAMESPACE.clone(),
         vec![LOCAL_KMS.clone()],
     )?
+    .extra_options(EXTRA_OPTIONS.clone())
```

Comment:
Without this (and the change to the v2 runner), the tests were accidentally using mongocryptd instead of crypt_shared.